.github/workflows/build.yml

```diff
@@ -159,3 +159,29 @@ jobs:
       - name: "Perform quality checks: pylint_srctest"
         run: |
           temp/bin/tox -e pylint_srctest
+
+  publish:
+    name: "PyPi Publish"
+    needs: ["test", "quality"]
+    if: ${{ ! startsWith(github.ref, 'refs/tags/') }}
```

Comment:
Why is the github action run (and fail) if it should run only when a new tag is pushed?
Maybe this check is incorrect.